src/importexport/midi/internal/midiconfiguration.cpp

```diff
@@ -34,12 +34,17 @@ using namespace mu::iex::midi;
 static const Settings::Key SHORTEST_NOTE_KEY("iex_midi", "io/midi/shortestNote");
 static const Settings::Key EXPORTRPNS_KEY("iex_midi", "io/midi/exportRPNs");
 static const Settings::Key EXPAND_REPEATS_KEY("iex_midi", "io/midi/expandRepeats");
+static const Settings::Key PEDAL_EVENTS_MIN_TICKS_KEY("iex_midi", "io/midi/pedalEventsMinTicks");
 
 void MidiConfiguration::init()
 {
     settings()->setDefaultValue(SHORTEST_NOTE_KEY, Val(mu::engraving::Constants::DIVISION / 4));
     settings()->setDefaultValue(EXPAND_REPEATS_KEY, Val(true));
     settings()->setDefaultValue(EXPORTRPNS_KEY, Val(true));
+    settings()->setDefaultValue(PEDAL_EVENTS_MIN_TICKS_KEY, Val(1));
+    settings()->setDefaultValue(PEDAL_EVENTS_MIN_TICKS_KEY, Val(1));
+    settings()->setDescription(PEDAL_EVENTS_MIN_TICKS_KEY, muse::qtrc("project/export", "Min. tick value for pedal events").toStdString());
```

Comment:
```suggestion

    settings()->setDefaultValue(PEDAL_EVENTS_MIN_TICKS_KEY, Val(1));
    settings()->setDescription(PEDAL_EVENTS_MIN_TICKS_KEY, muse::trc("iex_midi", "Min. tick value for pedal events"));
```
(Oops, sorry, this comment comes apparently _just_ too late to be combined with the rebase)

Reply:
Oops, a stupid copy/paste error.

Done